src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -446,36 +463,13 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
                 />
               }
               label={
-                <Box>
+                <div style={{ marginTop: '1.2em' }}>
```

Comment:
Can you use the box component here please?

We're also not really using units like em at the moment. Our design system is based on units of 8px. 

See if you can line things up with a box and the sx prop and a marginTop of x * 8px, where x is whatever looks aligned..